guidelines/terms/21/motion-animation.html

```diff
@@ -1,7 +1,5 @@
 <dt><dfn id="dfn-motion-animation">motion animation</dfn></dt>
 <dd>
-   					
    <p>addition of steps between conditions to create the illusion of movement or to give a sense of a smooth transition</p>
-	<aside class="example"><p>For example, an element which moves into place or changes size while appearing is considered to be animated. An element which appears instantly without transitioning is not using animation. Motion animation does not include changes of color, blurring, or opacity which do not change the perceived size, shape, or position of the element.</p></aside>
-   				
+	<aside class="example"><p>For example, an element which moves into place or changes size while appearing is considered to be animated. An element which appears or changes instantly, without transition/animation steps, is not using animation.</p><p>Motion animation does not include changes – such as changes of color or opacity – that do not alter the perceived size, shape, position, or distance/depth of the element.</p></aside>
```

Comment:
```suggestion
	<aside class="example"><p>For example, an element which moves into place or changes size while appearing is considered to be animated. An element which appears or changes instantly, without transition/animation steps, is not using animation.</p><p>Motion animation does not include changes – such as changes of color or opacity – that do not alter the perceived size, shape, position, or distance/depth of the element to the viewer.</p></aside>
```

Reply:
i'll admit, that reads a bit stilted to me. isn't it already implicit in the fact that this talks about "perceived..." as it's clearly being perceived by a user/viewer?

Reply:
Agree, the use of perceived makes "to the viewer" redundant.

Reply:
In the context of cookiecrook's comment: "this is critical in the context of emerging modalities like XR, in which motion may be perceivable from another observable camera angle, but not the primary."
I can see why it isn't as redundant as it appears from just the code change.